Constuctor Exercises.js

```diff
@@ -0,0 +1,250 @@
+
+let freezingPoint = {celsius: 0 , fahrenheit: 32, kelvin: -273}
+
+function Celsius(celsius) {
+    this.celsius = celsius;
+}
+Celsius.prototype.getFahrenheitTemp = function(){
+    return this.getFharenheitTemp = 1.8 * this.celsius + 32
+}
+Celsius.prototype.getKelvinTemp = function() {
+    return this.getKelvinTemp = this.celsius + 273;
+}
+
+Celsius.prototype.isBelowFreezing = function() {
+    if(this.celsius <= 0) {
+        return true
+    } else {
+        return false
+    }
+}
+
+let outsideTempt = new Celsius(10.0)
+console.log(outsideTempt.celsius) //returns 10.0
+console.log(outsideTempt.getKelvinTemp()) //returns 283.0
+console.log(outsideTempt.getFahrenheitTemp()) //returns 50.0
+
+// ## Question 5
+// a. Create a constructor function called `Movie` that has properties for 
+// `name`, `year`, `genre`, `cast`, and `description`. Create an instance of your 
+// `Movie`
+
+// // b. Create an prototype function inside `Movie` called `blurb` that returns a 
+// formatted string describing the movie.
+
+// // Ex: "Borat came out in 2006. It was an odd film starring Sacha Baron Cohen as 
+// a man named Borat who was visiting America from Kazakhstan."
+
+function Movie(name, year, genre, cast, description) {
+    this.name = name;
+    this.year = year;
+    this.genre = genre;
+    this.cast = cast;
+    this.description = description;
+
+    }
+let borat = new Movie("Borat", 2006, "Odd Film", "Sacha Baron Cohen", "a man named Borat who was visiting America from Kazakhstan.");
+
+Movie.prototype.blurb = function() {
+    return this.blurb = (this.name + " " + this.year + " " + this.genre +
+    " " + this.cast + " " + this.description)
+}
+console.log(borat.blurb());
+
+
+// ## Question 6
+ 
+// Write a constructor Vector that represents a vector in two-dimensional space.
+// It takes two number arguments: `x` and `y` parameters, which it should be saved to 
+// properties of the same name.
+
+// Give the Vector prototype two methods, `plus` and `minus`, that take another vector as an
+//  argument and
+// returns a new vector that has the sum or difference of the two vectors’ 
+// (the one in `this` and the parameter) x and y values.
+
+// Add a method `getLength` to the prototype that computes the length of the vector ;
+// that is, the distance of the point (x, y) from the origin (0, 0).(a^2 + b^2 = c^2)
+
+// [Vectors at mathisfun.com](https://www.mathsisfun.com/algebra/vectors.html)
+
+// ```js
+// var v1 = new Vector(1, 2)
+// var v2 = new Vector(2, 3)
+// console.log(v1.plus(v2));
+// // => Vector {x: 3, y: 5}
+// console.log(v1.minus(v2));
+// // => Vector {x: -1, y: -1}
+
+// var v3 = new Vector(3, 4)
+// console.log(v3.getLength());
+// // => 5
+// ```
+
+function Vector (X, Y) {
+    this.X = X;
+    this.Y = Y;
+
+}
+Vector.prototype.plus = function(vector) {
+ let sumX = this.X + this.Y
+ let sumY = this.Y + this.X
+ let newvector = new Vector(sumX, sumY); 
+ return newVector
```

Comment:
Here you need to use the vector in the argument. 
Ex:` let sumX = this.x + vector.x`